docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md

```diff
@@ -239,112 +255,123 @@ running the following commands on **Command Prompt (Run as Adminstrator)**:
 
-    ![Create Volume](images/create_volume.png)
+Once successfully Volume is created, we can use the Volume to launch an instance
+as shown below:
 
-    Once successfully Volume is created, we can use the Volume to launch an instance
-    as shown below:
+![Launch Instance from Volume](images/launch_instance_from_volume.png)
 
-    ![Launch Instance from Volume](images/launch_instance_from_volume.png)
+Add other information and setup a Security Group that allows RDP as shown below:
 
-    Add other information and setup a Security Group that allows RDP as shown below:
+![Launch Instance Security Group for RDP](images/security_group_for_rdp.png)
 
-    ![Launch Instance Security Group for RDP](images/security_group_for_rdp.png)
+![Running Windows Instance](images/win2k22_instance_running.png)
 
-    ![Running Windows Instance](images/win2k22_instance_running.png)
+![Associate Floating IP](images/win_instance_add_floating_ip.png)
 
-    ![Associate Floating IP](images/win_instance_add_floating_ip.png)
+Click on detail view of the Instance and then click on Console tab menu
+and click on **"Send CtrlAltDel"** button located on the top right side of
+the console as shown below:
 
-    Click on detail view of the Instance and then click on Console tab menu
-    and click on **"Send CtrlAltDel"** button located on the top right side of
-    the console as shown below:
+![View Console of Instance](images/console_win_instance.png)
 
-    ![View Console of Instance](images/console_win_instance.png)
+![Administrator Sign in Prompt](images/administrator_singin_prompt.png)
 
-    ![Administrator Sign in Prompt](images/administrator_singin_prompt.png)
+![Administrator Prompted to Change Password](images/ok_to_change_password_administrator.png)
 
-    ![Administrator Prompted to Change Password](images/ok_to_change_password_administrator.png)
+![Set Administrator Password](images/new_password_administrator.png)
 
-    ![Set Administrator Password](images/new_password_administrator.png)
+![Proceed Changed Administrator Password](images/proceed_change_password_administrator.png)
 
-    ![Proceed Changed Administrator Password](images/proceed_change_password_administrator.png)
+![Administrator Password Changed Successful](images/password_changed_success.png)
 
-    ![Administrator Password Changed Successful](images/password_changed_success.png)
+### 14. How to have Remote Desktop login to your Windows instance
 
-14. Remote Desktop login should work with the FLoating IP associated with the instance:
+Remote Desktop login should work with the Floating IP associated with the instance:
 
-    ![Search Remote Desktop Protocol locally](images/RDP_on_local_machine.png)
+![Search Remote Desktop Protocol locally](images/RDP_on_local_machine.png)
 
-    ![Connect to Remote Instance using Floating IP](images/remote_connection_floating_ip.png)
+![Connect to Remote Instance using Floating IP](images/remote_connection_floating_ip.png)
```

Comment:
Why are we showing the user a failed attempt to log in?

Reply:
I am uploading a new screenshot for this now.